In [1]:
import numpy as np
from scipy.stats import t

In [2]:
# Given data
sample_mean = 3050  # sample mean weekly cost (Rs.)
theoretical_mean = 1000 + 5 * 600  # theoretical mean weekly cost according to the cost model (W = $1,000 + $5X for X = 600 units)
standard_deviation = 5 * 25  # standard deviation of the population
sample_size = 25  # sample size
alpha = 0.05  # significance level

In [3]:
# Calculate the test statistic (t)
standard_error = standard_deviation / np.sqrt(sample_size)
t_statistic = (sample_mean - theoretical_mean) / standard_error

In [4]:
print("standard error",standard_error)
print("t statistic",t_statistic)

standard error 25.0
t statistic -38.0


In [5]:
# Determine the critical value from the t-distribution
degrees_of_freedom = sample_size - 1
critical_value = t.ppf(1 - alpha, degrees_of_freedom)

In [6]:
critical_value

1.7108820799094275

### conclusion

Since the t-statistic (-38.0) is far less than the critical value (-1.71 for a left-tailed test or ±1.71 for a two-tailed test), we reject the null hypothesis. This indicates that the sample mean weekly cost (3050) is significantly different from the theoretical mean weekly cost (1000+5×600=
4000)